app/views/admin/booths/index.html.haml

```diff
@@ -0,0 +1,58 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        Booths
+        = "(#{@booths.length})" if @booths.any?
+        .pull-right
+          - if can? :create, Booth
+            = link_to 'Add Booth', new_admin_conference_booth_path(@conference.short_title), class: 'button btn btn-primary'
+      %p.text-muted
+        All the booth requests
+.row
+  .col-md-12
+    .margin-booth-table
+      %table.table.table-striped.table-bordered.table-hover.datatable
+        %thead
+          %th
+            %b ID
+          %th
+            %b Logo
+          %th
+            %b Title
+          %th
+            %b Submitter
+          %th
+            %b Responsibles
+          %th
+            %b State
+          %th
+            %b Actions
+        - @booths.each do |booth|
+          %tr
+            %td
+              = booth.id
+            %td
+              = image_tag(booth.picture.thumb.url, width: '20%')
+            %td
+              = link_to booth.title, admin_conference_booth_path(@conference.short_title, booth)
+            %td
+              = link_to booth.submitter.name, admin_user_path(booth.submitter) if booth.submitter
+            %td
+              .responsibles
+                - booth.responsibles.each do |responsible|
```

Comment:
Let's make this look a bit nice, ie separate users with comma. We already have a function to do that for speakers, perhaps it's worth using it?